e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-submit.spec.js

```diff
@@ -0,0 +1,55 @@
+import { companyDetails } from '../../../../pages/your-business';
+import { continueButton } from '../../../../pages/shared';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import partials from '../../../../partials';
+import { ROUTES, FIELD_IDS } from '../../../../../../constants';
+
+const {
+  EXPORTER_BUSINESS: {
+    YOUR_COMPANY: {
+      TRADING_NAME,
+    },
+  },
+} = FIELD_IDS.INSURANCE;
+
+const COMPANY_DETAILS_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+
+describe('Your business - company house search - As an Exporter I want to enter details about my business in \'your business\' section', () => {
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.YOUR_BUSINESS.COMPANY_DETAILS, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
```

Comment:
Can we please use the cypress command to complete the full user journey and test the URL with reference number - will save us time when we introduce login functionality

```js
cy.submitInsuranceEligibilityAndStartApplication();

taskList.prepareApplication.tasks.policyTypeAndExports.link().click();

getReferenceNumber().then((id) => {
  referenceNumber = id;

  cy.url().should('eq', expected);
});
```

Example here https://github.com/UK-Export-Finance/exip/blob/main-application/e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/type-of-policy.spec.js

In all E2E tests

Reply:
discussed, no url functionality yet for reference number for your-business